Error running with React 15.4

If you have react@15.4.0-rc.4 installed redbox-react will fail with a file not found error.

    ERROR in ./~/redbox-react/~/react-dom/index.js
    Module not found: Error: Can't resolve 'react/lib/ReactDOM'

Redbox-react installs 15.3 which depends on the project level React version, 15.4 has changed the structure and this file is now in ReactDOM itself.

--- a/package.json
+++ b/package.json
@@ -51,11 +51,11 @@
     "webpack": "^1.13.1"
   },
   "peerDependencies": {
-    "react": "^0.14.0 || ^15.0.0"
+    "react": "^0.14.0 || ^15.0.0",
+    "react-dom": "^0.14.0 || ^15.0.0"
   },
   "dependencies": {
     "error-stack-parser": "^1.3.6",
-    "object-assign": "^4.0.1",
-    "react-dom": "^0.14.0 || ^15.0.0"
+    "object-assign": "^4.0.1"
   }
 }

